### ISSUE TITLE
3ds Max Export: Pivot transformation not taken into account with mesh export

### ISSUE THREAD

**Issue by PierreDuc** [BabylonJS/Babylon.js#597]
In 3ds Max you can adjust the pivot only. For instance placing the pivot in the middle of the object after 'sculpting', so the rotation goes well.

Unfortunately this transformation is not applied in the export of the mesh. I know where to get the values, just don't know how to apply it in the export.

`meshNode.MaxNode.ObjOffsetPos;`
`meshNode.MaxNode.ObjOffsetRot;`
`meshNode.MaxNode.ObjOffsetScale;`

**Comment by deltakosh** [comment]
Hello, you can update the exporter to save the pivot matrix (after changing it to left handed) to .pivotMatrix

This is where the loader loads it:
https://github.com/BabylonJS/Babylon.js/blob/master/src/Loading/Plugins/babylon.babylonFileLoader.ts#L693

You should add it around here:
https://github.com/BabylonJS/Babylon.js/blob/master/Exporters/3ds%20Max/Max2Babylon/Exporter/BabylonExporter.Mesh.cs#L209

**Comment by PierreDuc** [comment]
Thanks for the reply!

I already tried your suggestion (sort of). The `ObjOffset` values are, as the name suggests, offset values. Unfortunately I'm not sure how these values relate to the object

I did figure out that the `ObjOffsetPos` should be multiplied by the `ObjOffsetScale`. But that's about it. Isn't it possible to get the actual position in the exported file as shown in 3ds Max? (this includes the transformation on the pivot)
